src/external/rawspeed/data/cameras.xml

```diff
@@ -5092,8 +5092,30 @@
 			<Color x="0" y="1">GREEN</Color>
 			<Color x="1" y="1">RED</Color>
 		</CFA>
-		<Crop x="20" y="18" width="-14" height="-5"/>
-		<Sensor black="0" white="65535"/>
```

Comment:
Looking at the origin of the entry for that camera+mode, i guess it was added without sample